tests/scripts/gen_release_notes.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -e
+
+function help() {
+  print="
+  suppose we want release note for v1.6.8, we'll check the list of pr merged in upstream/release-1.6 but not present in v1.6.7
+  EXAMPLE command:
+  sh tests/scripts/gen_release_notes.sh <rook/rook remote name>/release-1.6 v1.6.7
+  "
+  echo "$print"
+}
+
+trap help EXIT
```

Comment:
We don't need to show the help for any errors.

Reply:
okay